src/encoded/tests/test_audit_dataset.py

```diff
@@ -1,6 +1,8 @@
 import pytest
 
 
+# References are required and set to have at least one item, no empty lists are allowed
```

Comment:
if this test is no longer going to be needed because a dependency was introduced you should delete the test, and add another test in the schema testing file to test your dependency and schema validation